local-installation/separate-scripts/start-distributed-analytics.cmd

```diff
@@ -1,7 +1,16 @@
 @ECHO off
 
-SET TAG=2.1.4
+SET TAG=2.2.0
 SET REGISTRY=harbor.honeur.org
 
+echo -n "Enter organization name: "
```

Comment:
Should become something like:

set "db_password="
set /p "db_password=DB password: "